homeworks/03-data-structure/src/BinarySearchTree.test.ts

```diff
@@ -0,0 +1,37 @@
+import {BinarySearchTree} from "./BinarySearchTree"
+
+describe("Binary Search Tree", () => {
+    const tree = new BinarySearchTree({
+        value: 20,
+        left: {
+            value: 15,
+            left: { value: 13 },
+            right: { value: 18 }
+        },
+        right: {
+            value: 25,
+            right: { value: 30 }
+        }
+    });
+
+    describe("Search elements by 'has' function", () => {
+       it("Search for existing max value element", () => {
+           expect(tree.has(30)).toBeTruthy();

```

Comment:
`toBeTrue()` is more correct. The test will be passed even if `tree.has(30)` return `99`.